{
  "comments": [
    {
      "key": {
        "uuid": "b9a9736d_604bb780",
        "filename": "src/config/common/cfgm_common/db_json_exim.py",
        "patchSetId": 1
      },
      "lineNbr": 229,
      "author": {
        "id": 40
      },
      "writtenOn": "2020-08-24T06:41:33Z",
      "side": 1,
      "message": "How this can occur, can you add a comment here explaining the reason that the env is returning NoNode and why we can safely continue?",
      "revId": "5ac36820435f02fc773f0d2710791e5cf4a045e9",
      "serverId": "c72cd27d-1743-4a55-84a4-cd13b8400c52",
      "unresolved": true
    }
  ]
}